modules/querier/config.go

```diff
@@ -84,6 +87,7 @@ func (cfg *Config) RegisterFlagsAndApplyDefaults(prefix string, f *flag.FlagSet)
 		},
 		DNSLookupPeriod: 10 * time.Second,
 	}
+	cfg.ShuffleShardingLookbackPeriod = 1 * time.Hour
```

Comment:
we should be set to `query_ingesters_until` and some buffer for safety...

Reply:
Yep, I looked at this parameter. `query_ingesters_until` defaults to 30m so I thought 1hr would be safe. We can also default to a slightly higher value, the performance impact is small.